components/sysutils/nmap/Makefile

```diff
@@ -54,29 +52,29 @@ COMPONENT_PREP_ACTION = (cd $(@D)/libdnet-stripped && autoconf)
 
 COMPONENT_PRE_CONFIGURE_ACTION = $(CP) -rp $(SOURCE_DIR)/. $(@D)
 
-CONFIGURE_OPTIONS  +=   CPPFLAGS="$(CPPFLAGS)"
+CONFIGURE_OPTIONS  +=   CPPFLAGS="-I$(USRINCDIR)/pcre"
+CONFIGURE_OPTIONS  +=   LDFLAGS="-L$(OPENSSL_LIBDIR.64)"
```

Comment:
Is this really needed with `--with-openssl=$(OPENSSL_PREFIX)` below?

Reply:
Yes, because otherwise it picks up the 32 bit library instead.

Reply:
I see.  _(rant on)_ If they would use pkgconfig then we would not need to do this at all _(rant off)_.  Anyway, I assume you can safely drop `.64` from this line.  Also, please add short comment above this line explaining why we do need it (something like `# otherwise it picks up the 32 bit library instead` would be okay).  Thank you.

Reply:
done